docs/GETTING_STARTED.md

```diff
@@ -67,11 +65,9 @@ Create namespace `local-ozone`, and deploy Ozone S3:
 
     $ sh ./setup-ozone-s3.sh
 
-This can take a few minutes. Ensure all the pods are running before moving on to the next step.
-
-    $ kubectl -n local-ozone get pods
+This can take a few minutes. Ensure all the pods are running for 1 minute before moving on to the next step.
 
-> Up to today [Ozone does not support arm64 (Apple Silicon) well](https://issues.apache.org/jira/browse/HDDS-6263). If you are using Apple Silicon and want to set up dev services locally, you can choose to use an Intel machine, or use a remote K8s cluster / S3 endpoint as an alternative.
+    $ watch -n 5 kubectl -n local-ozone get pods
```

Comment:
the `watch` here is a prerequisite. please either remove this or add it to the prerequisite on the top.